authentication/middleware.py

```diff
@@ -22,7 +22,7 @@ class HttpOnlyTokenAuthentication(BaseAuthentication):
     def get_model(self):
         if self.model is not None:
             return self.model
-        from rest_framework.authtoken.models import Token
```

Comment:
I think this got automatically created when I was playing around with Github co-pilot.